docs/openstack-glance-images.md

```diff
@@ -390,3 +390,26 @@ openstack --os-cloud default image create \
           --property os_version=9.4 \
           RHEL-9.4
 ```
+
+## Get Windows
+
+!!! note
+
+    You will need to create a virtual disk image from your own licensed media and convert to .qcow2 format.  This example uses a Windows 2022 Standard Edition installation generalized with cloud-init and sysprep, then converted the image to .qcow2 format using qemu-img.  For additional information on creating a Windows image, please see the upstream documentation https://docs.openstack.org/image-guide/create-images-manually-example-windows-image.html.
```

Comment:
Minor change refer documentation like this  [upstream documentation](https://docs.openstack.org/image-guide/create-images-manually-example-windows-image.html)